lib/linters/i18n_helper_html_linter.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module IdentityIdp
+      # This linter checks to ensure that strings which include HTML are rendered using Rails `t`
+      # view helper, rather than through the I18n class. Only the Rails view helper will mark the
+      # content as HTML-safe.
+      #
+      # @see https://guides.rubyonrails.org/i18n.html#using-safe-html-translations
+      #
+      # @example
+      #   # bad
+      #   I18n.t('errors.message_html')
+      #
+      #   # good
+      #   t('errors.message_html')
+      #
+      class I18nHelperHtmlLinter < RuboCop::Cop::Base
+        MSG = 'Use the Rails `t` view helper for HTML-safe strings'
+
+        RESTRICT_ON_SEND = [:t].freeze
+
+        def_node_matcher :i18n_class_send?, <<~PATTERN
+          (send (const nil? :I18n) :t $...)
+        PATTERN
+
+        def on_send(node)
+          return if !i18n_class_send?(node) || !i18n_key(node)&.end_with?('_html')
```

Comment:
Love this linter! Is there a way we can check like `node.filename` to confirm we're in an ERB?

Reply:
While it's currently limited to linting ERB files, I think _ideally_ we could lint every file. Not sure how realistic it'd be to enforce this everywhere, since PORO classes won't easily have access to the view helper, but I could at least imagine broadening to include controllers for example.

Reply:
Oh right, controllers, makes sense

Reply:
On that note, I added controllers to the list of included file patterns in 54055a2488.